src/subgraph/abs.c

```diff
@@ -192,5 +224,7 @@ enum xnn_status xnn_define_abs(
   node->reshape = reshape_abs_operator;
   node->setup = setup_abs_operator;
 
+  node->infer_shape_forward = infer_shape_forward;
```

Comment:
As we discussed, and putting it here just so I remember, making this a separate function (as opposed to output shape calculated with in reshape) may lead to some redundant calculations i.e. [conv outputs](https://github.com/google/XNNPACK/blob/00b05588e0b43b74ad2ae0526cbaac507837d4a0/src/subgraph/convolution-2d.c#L358)

Reply:
This is no longer relevant. Everything is done in reshape